bloptools/gp/__init__.py

```diff
@@ -421,6 +421,7 @@ def update(self, reuse_hypers=True, verbose=False):
         self.validator.train(training_iter=self.training_iter, reuse_hypers=reuse_hypers, verbose=verbose)
 
     def acquire_with_bluesky(self, params, routing=True, verbose=False):
+        
```

Comment:
This line causes linting/pre-commit workflow errors. Please remove it.

❗Also, please make sure you have `pre-commit install` command run in the repo's root. This will make sure the linter runs before the commit is produced.